### ISSUE TITLE
Step 0: client/app.ts (5, 22): Cannot find module './app.html'.

### ISSUE THREAD

**Issue by ChristophRob** [Urigo/angular-meteor-docs#190]
After Step 0 my terminal always shows this error. It happens with every template through the tutorial which ist loaded as `import template from './app.html';`
The app is working fine, but i don't understand why the error shows up

```
$ meteor
=> Started proxy.                             
=> Started MongoDB.                           
client/app.ts (5, 22): Cannot find module './app.html'.
=> Started your app.                          

=> App running at: http://localhost:3000/ 
```

**Comment by artmadeit** [comment]
Hi!
The same problem is on:
https://github.com/Urigo/meteor-angular2.0-socially/issues/116
and
https://github.com/Urigo/angular-meteor-docs/issues/148

It should be fixed by Urigo/meteor-static-templates#4 as @kamilkisiela mention.

**Comment by kamilkisiela** [comment]
@Urigo It should be fixed with the new version of Socially 2.0

**Comment by Urigo** [comment]
closing, please open a new issue if the problem still happens
